Student/TechDebt/Files and Folders/FileViewController.m

```diff
@@ -98,7 +98,7 @@ - (void)viewDidLoad {
 
     [self updateForURLState];
     
-    [self.view setBackgroundColor:[UIColor colorNamed:@"backgroundLightest" inBundle:NSBundle.core compatibleWithTraitCollection:nil]];
```

Comment:
This was returning `nil`. I couldn't figure out how to get the color using `colorNamed:inBundle:` so I added an `@objc` helper to Core.